src/SSW.Rewards.Infrastructure/Persistence/Migrations/20220826054420_AddQuizAttemptRecord.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Remove this migration and re-add it once the relationships etc are fixed.

Reply:
Done